src/ui/components/address-action/RecipientLine/RecipientLine.tsx

```diff
@@ -27,7 +32,24 @@ export function RecipientLine({
     <Surface style={{ borderRadius: 8, padding: '10px 12px' }}>
       <Media
         image={
-          <BlockieImg address={checksumAddress} size={36} borderRadius={6} />
+          <div
+            style={{
+              position: 'relative',
+              width: 36,
```

Comment:
I think, we can remove this size style. Inner element will handle it anyway.

Reply:
✅